projects/showcase/cypress/e2e/popup.cy.ts

```diff
@@ -1,26 +1,31 @@
-import { E2eDriver } from '../support/e2e-driver';
+import { E2eDriver } from "../support/e2e-driver";
```

Comment:
@ShellyDCMS please run the linter again, seems like not all files are using the same rules on quotes, right?